go/vt/topo/helpers/tee.go

```diff
@@ -187,6 +187,17 @@ type teeTopoLockDescriptor struct {
 
 // Lock is part of the topo.Conn interface.
 func (c *TeeConn) Lock(ctx context.Context, dirPath, contents string) (topo.LockDescriptor, error) {
+	return c.lock(ctx, dirPath, contents)
+}
+
+// TryLock is part of the topo.Conn interface.
+// As of today it provides same functionality as Lock
```

Comment:
Nit, but we are not consistently using sentence punctuation throughout such as here.